addons/mrp/models/stock_move.py

```diff
@@ -223,6 +223,14 @@ def _generate_move_phantom(self, bom_line, product_qty, quantity_done):
             return move
         return self.env['stock.move']
 
+    def _get_document(self):
```

Comment:
try to merge in _get_upstream_documents_and_responsibles
(rename _get_upstream_documents_and_responsibles to _get_document_responsible with 2 kwargs for upstream=False and upstream_documents_visited=False)